.github/CODEOWNERS

```diff
@@ -48,7 +48,7 @@ llvm/ @intel/dpcpp-tools-reviewers
 clang/tools/clang-offload-*/ @intel/dpcpp-tools-reviewers
 
 # Explicit SIMD
-SYCLLowerIR/ @intel/dpcpp-esimd-reviewers
+ESIMD/ @intel/dpcpp-esimd-reviewers
```

Comment:
```suggestion
ESIMD/ @intel/dpcpp-esimd-reviewers
SYCLLowerIR/LowerESIMD* @intel/dpcpp-esimd-reviewers
SYCLLowerIR/ESIMD* @intel/dpcpp-esimd-reviewers
```

Reply:
If I understand correctly, the benefit of `ESIMD` is that it matches `ESIMD` directory anywhere in the repo, which will also automatically cover tests, for example

Reply:
Right. 
@kbobrovs, are new rules you suggest match new files not matched already? If so, can we move them to ESIMD directory?

Reply:
@bader, I wasn't aware ESIMD/ will match all the cases I added, so, I approve